opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -123,7 +125,7 @@ namespace Opm {
 
 
         const UnitSystem& m_deckUnitSystem;
-        const ParseContext& m_parseContext;
+        const ParseContext m_parseContext;
```

Comment:
That's true, moves are simply pointer-copies that are "marked" for non-delete when the old host object goes out of scope. But if an object doesn't point to any larger blobs of memory it's typically tiny anyway, meaning copies aren't expensive because most objects only contain a handful of tiny things (typically numbers). Using a reference-then-copy doesn't change that, and also doesn't enable copy elision.

And at the end of the day we're going to copy here sooner or later, and if that is done in the constructor argument some opportunities are enabled, but in the worse case we should have no difference compared to reference-then-copy.